asyncio_run_in_process/_utils.py

```diff
@@ -0,0 +1,50 @@
+import io
+import struct
+import sys
+from typing import Any, BinaryIO, Tuple
+
+import cloudpickle
+
+
+def get_subprocess_command(
+    child_r: int, child_w: int, parent_pid: int
+) -> Tuple[str, ...]:
+    from . import _child
+
+    return (
+        sys.executable,
+        "-m",
+        _child.__name__,
+        "--parent-pid",
+        str(parent_pid),
+        "--fd-read",
+        str(child_r),
+        "--fd-write",
+        str(child_w),
+    )
+
+
+def pickle_value(value: Any) -> bytes:
+    serialized_value = cloudpickle.dumps(value)
+    # mypy thinks this is returning an `Any` type.
+    return struct.pack(">I", len(serialized_value)) + serialized_value  # type: ignore
```

Comment:
Are you avoiding `cast` for performance reasons?

Reply:
No, the ignore was just the first thing I though of.  I've updated so that this isn't needed now.